feature_apps/build.gradle.kts

```diff
@@ -12,5 +12,6 @@ android {
 dependencies {
   implementation(project(ModuleDependency.APTOIDE_NETWORK))
   implementation(project(ModuleDependency.APTOIDE_UI))
+  implementation(project(ModuleDependency.FEATURE_APPCOINS))
```

Comment:
Like i said before i dont know if the bonus view justifies having a whole separate module

Reply:
Yes, it is better to keep app coins related staff in a dedicated module